src/main/java/com/exactpro/th2/codec/csv/CsvCodec.java

```diff
@@ -72,121 +77,134 @@ public CsvCodec(MessageRouter<MessageBatch> router, MessageRouter<EventBatch> ev
         this.defaultHeader = defaultHeader == null
                 ? null
                 : defaultHeader.stream().map(String::trim).toArray(String[]::new);
+        if (this.defaultHeader != null && this.defaultHeader.length == 0) {
```

Comment:
ArrayUtils.isEmpty(this.defaultHeader)

Reply:
We cannot do that because we don't need to throw an error in case the defaultHeader is null